hack/local-down-karmada.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The script to remove the karmada started by hack/local-up-karmada.sh
+# example:
+# 1.hack/local-down-kaarmada.sh
+# 2.hack/local-down-kaarmada.sh -y ('-y' mean remove local images)
+
+clean_image=${1:-""}
+
+#step1 remove kind clusters
+clusters=(
+"${HOST_CLUSTER_NAME:-"karmada-host"}"
+"${MEMBER_CLUSTER_1_NAME:-"member1"}"
+"${MEMBER_CLUSTER_1_NAME:-"member2"}"
+"${MEMBER_CLUSTER_1_NAME:-"member3"}"
+)
+
+echo -e "\nStart remove kind cluster"
+for ((i=0;i<${#clusters[*]};i++))
+do
+  kind delete cluster --name ${clusters[i]}
+done
+echo "Remove kind clusters successful"
+
+#step2. remove kubeconfig
+echo -e "\nStart remove kubeconfig"
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+configs=(
+MAIN_KUBECONFIG=${MAIN_KUBECONFIG:-"${KUBECONFIG_PATH}/karmada.config"}
+MEMBER_CLUSTER_KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG:-"${KUBECONFIG_PATH}/members.config"}
+)
+for ((i=0;i<${#configs[*]};i++))
+do
+if [ -f ${configs[i]} ] ; then
+    rm ${MAIN_KUBECONFIG}
+      echo -e "Kubeconfig ${configs[i]} remove successful"
```

Comment:
```suggestion
    echo "Remove kubeconfig ${configs[i]} successfully."
```

`successfully` rather `successful`. The same mistake below.